frontend/src/components/admin/TemplateMailCard.vue

```diff
@@ -42,9 +42,12 @@ export default {
       required: true
     }
   },
+  data: () => ({
+    argsCount: 0
+  }),
   methods: {
     goToEditPage: function () {
-      // TODO
+      router.push({path: '/mailtemplate/edit/' + this.data.id})
     },
     deletePost: function () {
       RequestHandler.handle(EmailTemplateService.deleteEmailTemplateById(this.data.id))
```

Comment:
Misschien hier een location.reload() doen zo herlaad de pagina en is de template ook nadrukkelijk weg